libraries/exoplayer/src/main/java/androidx/media3/exoplayer/audio/AudioCapabilities.java

```diff
@@ -393,5 +404,20 @@ public static int getMaxSupportedChannelCountForPassthrough(
       }
       return 0;
     }
+
+    /**
+     * Returns an array list of surround encodings that maybe supported.
+     */
+    private static ImmutableList<Integer> getAllSurroundEncodingsMaybeSupported() {
```

Comment:
Ok, I see we will convert the return type to an array whenever we use it. 

Probably we can return the array type `int[]` here, so at the end of this method we do 
```
return Ints.toArray(encodings.build());
```
instead of 
```
return encodings.build();
```
Sorry for the back and forth!

Reply:
@tianyif  No worries! please see d4f8b7f7ed1cae89d725dfbf4d74492d3808356a  Thanks